cmd/main.go

```diff
@@ -690,7 +690,7 @@ func checkNeedsUpdate(ctx *Context, name string, compute string, destination str
 
 func copyToNode(ctx *Context, name string, compute string, destination string) error {
 	fmt.Printf("  Copying %s to %s at %s...", name, compute, destination)
-	if _, err := runCommand(ctx, exec.Command("scp", "-C", name, compute+":"+destination)); err != nil {
+	if _, err := runCommand(ctx, exec.Command("scp", "-OC", name, compute+":"+destination)); err != nil {
```

Comment:
What does `-OC` do?

Reply:
Use the legacy SCP protocol for file transfers instead of
             the SFTP protocol.  Forcing the use of the SCP protocol may
             be necessary for servers that do not implement SFTP, for
             backwards-compatibility for particular filename wildcard
             patterns and for expanding paths with a ‘~’ prefix for
             older SFTP servers.